marlowe-dashboard-client/README.md

```diff
@@ -1,83 +1,105 @@
-Marlowe Run (formerly called the Marlowe Dashboard) is an online simulator for Marlowe Applications.
+# Marlowe Run
```

Comment:
Lots of changes, probably better to review the rendered file: https://github.com/input-output-hk/marlowe-cardano/blob/hrajchert/scp-4015-improve-marlowe-run-dev-start-script/marlowe-dashboard-client/README.md

Reply:
Something that I didn't include on this PR but could be useful to add later is to visualize resources using:

```bash
htop -p $(pgrep cardano-node),$(pgrep marlowe-pab),$(pgrep cardano-wallet),$(pgrep chain-index),$(pgrep dashboard-server),$(pgrep webpack)
```